### DISCUSSION TITLE
Language is not showing up in repo

### DISCUSSION THREAD

**Discussion by yashera01** [github-linguist/linguist#6076] [Classification] [answered]
can anyone help me out ? I  have a  repo consisting css ,html and js ,but the language is not being detected in my repo
repo link:(https://github.com/yashera01/OIBSIP-Task3)

**Comment by lildude** [comment, marked as answer]
Your repo has no code in the default branch. Language analysis only happens against the default branch.

**Reply by yashera01** [reply]
Can you please tell how I can add my files to default branch which are currently in master Branch

**Reply by lildude** [reply]
Merge the [pull request](https://github.com/yashera01/OIBSIP-Task3/pull/1) I just created for you on your repo 😁 

What I did was:

- fork your repo by clicking the "Fork" button
- cloned it to my computer: `git clone https://github.com/lildude/OIBSIP-Task3.git`
- changed into the cloned directory: `cd OIBSIP-Task3`
- cherry-picked the only commit in your master branch: `git cherry-pick 15798ae`
- pushed the changes back to my fork of your repo: `git push origin main`
- opened a pull request

**Reply by yashera01** [reply]
yes ,it worked. thanks .
Can you plz tell me how can I do it on my own  because i am suffering with same issue in other repos

**Reply by lildude** [reply]
I just did. 😁 Perform steps 2-5 from above for each of your repos.
